packages/client-app/src/components/DataTable/DataTableInput.vue

```diff
@@ -94,3 +96,10 @@ const handleDropdownMouseUp = () => {
       @select="handleSelect" />
   </DataTableCell>
 </template>
+
+<style scoped>
+/* Tailwind placeholder is busted */
+input::placeholder {
```

Comment:
do we style it already from the theme reset style?

Reply:
I think so https://github.com/scalar/scalar/blob/main/packages/themes/src/reset.css#L62

Reply:
yea but didn't show up. You wanna work ur magic brynn? or we can just leave it, not the biggest thing 🤷🏾

Reply:
Oh weird, I'll look